code/modules/mob/living/taste.dm

```diff
@@ -25,11 +23,9 @@
 			text_output = pick("spiders","dreams","nightmares","the future","the past","victory",\
 			"defeat","pain","bliss","revenge","poison","time","space","death","life","truth","lies","justice","memory",\
 			"regrets","your soul","suffering","music","noise","blood","hunger","the american way")
-		if(text_output != last_taste_text || last_taste_time + 100 < world.time)
+		if((text_output != last_taste_text || last_taste_time + 100 < world.time) && !NO_TASTE_SENSITIVITY) // If you have no taste sensitivity, you get no message, because you can't taste it.
```

Comment:
comment entirely unnecessary, it explains something that is entirely evidence from the code.